cli/tests/integration/bundle_tests.rs

```diff
@@ -39,10 +41,10 @@ fn bundle_exports() {
     .output()
     .unwrap();
   // check the output of the test.ts program.
-  assert!(std::str::from_utf8(&output.stdout)
-    .unwrap()
-    .trim()
-    .ends_with("Hello"));
+  assert_ends_with!(
```

Comment:
Macros are used instead of functions because they say what line the test failed on.